### ISSUE TITLE
Error messages: "error: process ID out of range" and  "basename: missing operand" 

### ISSUE THREAD

**Issue by PicoSushi** [franciscolourenco/done#131]
Recently I'm trying **done**. 
The command `sleep 6` sometimes occurs error message as below:

```
error: process ID out of range

Usage:
 ps [options]

 Try 'ps --help <simple|list|output|threads|misc|all>'
  or 'ps --help <s|l|o|t|m|a>'
 for additional help text.

For more details see ps(1).
basename: missing operand
Try 'basename --help' for more information.
error: process ID out of range

Usage:
 ps [options]

 Try 'ps --help <simple|list|output|threads|misc|all>'
  or 'ps --help <s|l|o|t|m|a>'
 for additional help text.

For more details see ps(1).
```

These messages are sometimes shown, and sometimes not.
I guess that `$tmux_fish_ppid` is not working well.

my environment:

```
OS: Manjaro Linux x86_64 
Host: VirtualBox 1.2 
Kernel: 5.15.41-1-MANJARO 
Shell: fish 3.4.1 
Terminal: tmux 
```

**Comment by franciscolourenco** [comment]
@PicoSushi which command where/are you using to launch tmux? Are you using any options / parameters? Thanks!

**Comment by Feko** [comment]
Same thing here, seems like **done** comes as default in CachyOS. I'm launching tmux as:
```
        │ File: .config/fish/conf.d/tmux.fish
   1   │ if status is-interactive
   2   │     if not set -q TMUX
   3   │         tmux new-session -c $PWD
   4   │     end
   5   │ end
```

**Comment by franciscolourenco** [comment]
So the problem seems to be caused by the $PWD. I didn't expect this to happen since the v1.19.1.

@Feko could you check which version of done you have installed with the following command? 

```
echo $__done_version
```

**Comment by Feko** [comment]
Hey @franciscolourenco thanks for your quick response :slightly_smiling_face:  . I have:
```
❯ echo $__done_version
1.16.3
```
It seems the version that comes with CachyOS is somewhat outdated, I'll figure out how to update this.
Thanks again for your prompt reply!

**Comment by franciscolourenco** [comment]
@Feko can you confirm if upgrading fixed the problem?

**Comment by Feko** [comment]
Yup, updating locally fixed it :)

**Comment by franciscolourenco** [comment]
Fixed in v1.19.1. Please re-open if necessary.
